app/javascript/components/sponsor_us/SponsorCard.jsx

```diff
@@ -10,10 +10,13 @@ const SponsorCard = ({ type, amount, returns, icon }) => {
     return (
         <Card className="sponsor-card w-[22rem]">
             <div className="flex flex-row items-start">
-                <Icon className="h-14" />
+                <Icon className={type === 'Ruby' ? 'h-[58px] mt-[-5px]' : 'h-14'} />
```

Comment:
I added some conditional formatting here so that the Ruby gem is the same size/position as the other gems!